account_mail_templates/templates/account_mail_templates.xml

```diff
@@ -1,71 +1,49 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
 
-<!-- Adapt account.email_template_edi_invoice template -->
 <template
     id="mail_body_invoice_send"
     name="Invoice: Send by email"
 >
-    <t t-call="base_templates.mail_body_layout_basic">
-<<<<<<< HEAD
-        <t t-call="base_templates.msg_partner_refer">
-            <t t-set="partner" t-value="record.partner_id"/>
-        </t>
-        <t t-call="base_templates.msg_doc_description_with_amount">
-            <t t-set="doc_text">
-                <span name="span_doc_text">Here is your invoice</span>
+    <div style="margin: 0px; padding: 0px;">
+        <p style="margin: 0px; padding: 0px; font-size: 13px;">
+            Dear
+            <t t-if="object.partner_id.parent_id">
+                <t t-out="object.partner_id.name or ''">Brandon Freeman</t> (<t t-out="object.partner_id.parent_id.name or ''">Azure Interior</t>),
```

Comment:
Couldn't clause be used on main element?
```suggestion
                <t t-if="object.partner_id.parent_id" t-out="object.partner_id.name or ''">Brandon Freeman</t> (<t t-out="object.partner_id.parent_id.name or ''">Azure Interior</t>),
```

Reply:
This is mainly copy paste template from original.